File: spring/member-server/src/main/java/com/example/memberserver/infrastructure/IdProvier/aws/CognitoClient.java
package com.example.memberserver.infrastructure.IdProvier.aws;

import com.example.memberserver.infrastructure.IdProvier.IdpClient;
import com.example.memberserver.member.Member;

public class CognitoClient implements IdpClient {
    @Override
    public Long createMember(Member member) {
        throw new UnsupportedOperationException("Not yet implemented");
    }

    @Override
    public Long updateMember(Member member) {
        throw new UnsupportedOperationException("Not yet implemented");
    }

    @Override
    public void deleteMember(Member member) {
        throw new UnsupportedOperationException("Not yet implemented");
    }
}

File: spring/member-server/src/main/kotlin/com/example/memberserver/infrastructure/aws/CognitoClient.kt
package com.example.memberserver.infrastructure.aws

import com.example.memberserver.domain.member.IdpClient
import com.example.memberserver.domain.member.Member

class CognitoClient: IdpClient {
    override fun createMember(member: Member, password: String): Long? {
        TODO("Not yet implemented")
    }

    override fun updateMember(member: Member): Long? {
        TODO("Not yet implemented")
    }

    override fun deleteMember(member: Member) {
        TODO("Not yet implemented")
    }
}
